UI/data/themes/Acri.qss

```diff
@@ -719,11 +719,45 @@ MuteCheckBox::indicator:unchecked:disabled {
 	image: url(./Dark/unmute.png);
 }
 
+HeadphoneCheckBox {
+	outline: none;
+}
+
+HeadphoneCheckBox::indicator:checked {
+	image: url(./Dark/headphones_checked.png);
+}
+
+HeadphoneCheckBox::indicator:unchecked {
+	image: url(./Dark/headphones_unchecked.png);
+}
+
+HeadphoneCheckBox::indicator:unchecked:hover {
+	image: url(./Dark/headphones_unchecked.png);
+}
+
+HeadphoneCheckBox::indicator:unchecked:focus {
+	image: url(./Dark/headphones_unchecked.png);
+}
+HeadphoneCheckBox::indicator:checked:hover {
+	image: url(./Dark/headphones_checked.png);
+}
+
+HeadphoneCheckBox::indicator:checked:focus {
+	image: url(./Dark/headphones_checked.png);
+}
+
+HeadphoneCheckBox::indicator:checked:disabled {
+	image: url(./Dark/headphones_checked.png);
+}
+
+HeadphoneCheckBox::indicator:unchecked:disabled {
+	image: url(./Dark/headphones_unchecked.png);
+}
+
 OBSHotkeyLabel[hotkeyPairHover=true] {
 	color: rgba(27, 96, 166);
 }
 
-
```

Comment:
Probably don't need to make this change unless you were already changing the line functionally for some other reason.

Reply:
Updated to latest master.